### ISSUE TITLE
add service to systemd if not created yet

### ISSUE THREAD

**Issue by rprtr258** [rprtr258/pm#41]
so that after reboot pm can start processes which were saved using `pm save`

**Comment by rprtr258** [comment]
Managing system services with systemd has become relatively simple. You need to carefully watch that your service has sufficient privileges while still following the least privilege paradigm for security. Other than that you only need a *.service file in “/etc/systemd/system”.
```
# Service file under /etc/systemd/system/myserver.service
[Unit]
Description=My Server
After=network.target

[Service]
Type=simple
User=daemon
ExecStart=/usr/bin/myserver
Restart=on-failure

[Install]
WantedBy=multi-user.target
```
Once you’ve stored the service file in the path, you can run “sudo systemctl daemon-reload” to reload all daemon configurations and your service file. Afterwards you’ll be able to start your service with “sudo service myserver start”. With these relatively small additions to your Go application, you successfully daemonized your application already.

**Comment by rprtr258** [comment]
solved in 48830f15c790bbe318738f4d70c5cdd40c619c0f
